docs/virtual_machines/guest_agent_information.md

```diff
@@ -57,6 +58,11 @@ The infoSource field indicates where the info is gathered from. Valid values:
    - guest-agent: the info is based on Guest Agent report
    - domain, guest-agent: the info is based on both the domain spec and the Guest Agent report
 
+The queueCount field indicates how many queues were assigned to the interface (Multi Queue feature).
+Queue count value is derived from the domain XML.
```

Comment:
If the value is derived from the dom XML and not the ga, why should it be documented in the guest_agent_information doc?

Reply:
just because this is the only doc we are using to talk about the various fields of staus.interfaces

Reply:
Are there any other non ga fields here? If so maybe the doc title should be changed.

Reply:
Just the name

But the whole doc does mention GA a lot
maybe title can be about GA + about status.interfaces ? or something that combine them two in more general form ?
(file name as well)

Reply:
I would add queue count info to interfaces_and_networks doc, to the `virtio-net multiqueue` section. It doesn't fit here.